### DISCUSSION TITLE
"info hash is not authorized with this tracker"

### DISCUSSION THREAD

**Discussion by Coeur** [transmission/transmission#4546] [Q&A]
### What is the issue?

On a tracker, I get this warning:
>"info hash is not authorized with this tracker"

Is that something that can be circumvented, while still benefitting from this tracker?

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.0-beta.3.dev (cc4cbff049)

**Comment by ckerr** [comment]
>> "info hash is not authorized with this tracker"

> Is that something that can be circumvented, while still benefiting from this tracker?

I don't understand the question. If that torrent isn't allowed on that tracker, there's no way to trick the tracker to allowing it...?

**Reply by Coeur** [reply]
It was happening for all torrents. But now the warning is gone. Might have been a temporary issue with the tracker.

**Comment by josephj11** [comment]
I just upgraded from 3.x to 4.05. I have a number of trackers associated with each torrent.

My problem is that I can't (or don't know how) to) identify which tracker the message came from. I may have more than one with a red error message, but those messages are things like failed to connect... and don't say anything about an info hash...

Is there some way the message can indicate which tracker generated it?

**Reply by Coeur** [reply]
Can you give the exact message for which you want to know the tracker? Then I'll search the source code and see if we can improve on that.

**Reply by josephj11** [reply]
Thanks. The exact message is the title of this thread.

I may find a few others, but that's what I'm seeing most often.

**Reply by Coeur** [reply]
Ah, it's not a Transmission error message, it's an error message sent by the tracker, so it's not directly in the Transmission codebase. I'm not sure which part of the code is forwarding it to the log messages.

**Reply by AmISomeoneAtAll** [reply]
How do I fix it?

**Comment by josephj11** [comment]
@HomeroElCrack You can select the Trackers tab for that torrent, find the offending tracker (usually displayed with red messages), select it and press alt+R to remove that tracker from the torrent.

**Reply by AmISomeoneAtAll** [reply]
Thx! Although the issue surprisingly got fixed on its own!

**Reply by AmISomeoneAtAll** [reply]
Btw another question, what could be a good way to get more pairs in the torrent I'm downloading? My file is currently at 0 and I've pressed the "ask more pairs" button.

**Reply by josephj11** [reply]
IDK best practices, but here's what I do:
When I run a torrent
Open the Properties for a torrent (right click on the torrent)
Look at the trackers tab and see which ones are active (green)
Edit the trackers and copy and paste the good ones into a text file. I do this with a lot of torrents over time to get a good stable.
When I get a new torrent that has a no  trackers or ones that are not very good,
Edit the trackers and copy and paste any good ones that I haven't seen before into my text file
Select the whole text of the text  file and copy it
Select the whole trackers list in the torrent and hit del to delete it 
Paste in the list from my text file
Save the new trackers in the torrent
See which ones work and delete the ones that don't work - just for this torrent - not in my saved list

This gets me a set of torrents that's up to date and as good as it gets

----

When a torrent sits for a while, it starts looking for new peers every half hour or longer. When I think there's more  activity that I'm not getting, I select the torrents in the main list and right click and ask for more peers
Then I let it run for at least 10 minutes , possibly asking for more peers every 2 minutes or so if there isn't enough activity

If nothing is happening, I exit transmission, wait 10 seconds, and relaunch it. This gets all the trackers activated again whereas asking for more peers only gets some of  them.

This gets me much better performance. Just don't abuse it by doing it too frequently, overloading the trackers for nothing

----

I do almost all of my Internet activity from behind a VPN. It does not support port forwarding.

If your connection supports port forwarding, then setting it up correctly will make a huge difference in connections.
This lets the rest of the clients see that you're there and allows them to ask your client to connect. This can more than double your upload traffic and helps with downloads as well.

There are lots of posts and articles on how to do this and how to make sure it's working.

**Reply by AmISomeoneAtAll** [reply]
Thanks! The file finally got downloaded!
